features/named-color.yml

```
@@ -0,0 +1,7 @@
+name: Named colors
+description: The `named-color` CSS values are a standard set of colors that can be referenced by a name, such as `red` or `limegreen`.
```

Using the term `named-color` in a code block makes it seem (at least to me) like this is a keyword you can use when writing CSS.
Possible rephrase:

```suggestion
description: A standard set of colors that can be referenced by names, such as `red` or `limegreen`.
```